src/Microsoft.DncEng.DeployServiceFabricCluster/ClusterDeployer.cs

```diff
@@ -34,33 +35,59 @@ protected override void PopulateDefaultTags()
             DefaultTags["ClusterName"] = Settings.Name;
         }
 
-        protected override async Task<string> DeployResourcesAsync(List<IGenericResource> unexpectedResources, IAzure azure, IResourceManager resourceManager, CancellationToken cancellationToken)
+        protected override async Task<string> DeployResourcesAsync(
```

Comment:
It looks like there is some trailing spaces here. Its formatted weird on github.